Classes/SentryClient.php

```diff
@@ -77,22 +83,21 @@ public function initializeObject(): void
             return;
         }
 
-        $release = $this->settings['release'] ? $this->settings['release'] : '';
+        $release = $this->settings['release'] ?? '';
```

Comment:
We just changed this because the value might also be `false`. (Same is true for the other settings.)

Reply:
I think you forgot this one :)